xbmc/cores/RetroPlayer/RetroPlayer.cpp

```diff
@@ -258,8 +258,8 @@ bool CRetroPlayer::CloseFile(bool reopen /* = false */)
   m_gameClient.reset();
 
   m_renderManager.reset();
+  m_processInfo->ResetInfo();
```

Comment:
@garbear any reason you don't simply call `m_processInfo->Reset()` in `ResetInfo()` ?

Reply:
Probably because I wrote this in 2012 and Reset() was added in 2017. Can you change it?

Reply:
Also guard this with `if (m_processInfo)`. CloseFile() is called from the destructor to ensure consistent destruction order in case the external application fails to call CloseFile() for some reason. So CloseFile() could be called twice. Or remove the call from the destructor and ensure the application can't reset the player core without calling CloseFile().